package/extensions/default-extension.ts

```diff
@@ -209,6 +210,21 @@ export const defaultExtensions = (
   LinkPreview.configure({
     metadataProxyUrl: metadataProxyUrl,
   }),
+  ReminderBlock.configure({
+    onReminderCreate: (reminder) => {
+      // Handle reminder creation
+      console.log('reminder', reminder);
+    },
+    onReminderDelete: (reminderId) => {
+      // Handle reminder deletion
+      console.log('reminderId', reminderId);
+    },
+    onReminderUpdate: (reminder) => {
+      // Handle reminder updates
+      console.log('reminder', reminder);
+    },
+    reminders: [], // Existing reminders
```

Comment:
How would you get it from consumer ?

Since consumer will have all the business logic